client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/coverage/methodreplacement/classes/Base64DecoderClassReplacement.java

```diff
@@ -0,0 +1,51 @@
+package org.evomaster.client.java.instrumentation.coverage.methodreplacement.classes;
+
+import org.evomaster.client.java.instrumentation.coverage.methodreplacement.*;
+import org.evomaster.client.java.instrumentation.heuristic.Truthness;
+import org.evomaster.client.java.instrumentation.shared.ReplacementCategory;
+import org.evomaster.client.java.instrumentation.shared.ReplacementType;
+import org.evomaster.client.java.instrumentation.shared.StringSpecialization;
+import org.evomaster.client.java.instrumentation.shared.StringSpecializationInfo;
+import org.evomaster.client.java.instrumentation.staticstate.ExecutionTracer;
+
+import java.util.Base64;
+import java.util.Objects;
+
+public class Base64DecoderClassReplacement implements MethodReplacementClass {
+
+    @Override
+    public Class<?> getTargetClass() {
+        return Base64.Decoder.class;
+    }
+
+    private final static String BASE64_REGEX = "^[A-Za-z0-9+/]+={0,2}$";
```

Comment:
if this is coming from somewhere, best to put a link here as well

Reply:
I found a better regex for this. I am also adding the link to the source.